Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -209,6 +211,8 @@ class Error(OWWidget.Error):
         memory_error = Msg("Not enough memory.")
         no_class_values = Msg("Target variable has no values.")
         only_one_class_var_value = Msg("Target variable has only one value.")
+        test_data_incompatible = Msg(
+            "Test data may be incompatible with train data.")
```

Comment:
may? :)

Reply:
May. I don't want to be too categorical, it could already have had nan's before. But I can change "may be" to "is" if you think it's better. Your call.

Reply:
I did not consider this... Keep it "may".